src/components/Table/TasksTable.stories.tsx

```diff
@@ -1,5 +1,18 @@
+import { StoryObj } from "@storybook/react";
 import { TasksTable } from "./TasksTable";
 
+export default {
+  component: TasksTable,
```

Comment:
wondering somewhat if the `TasksTable` belongs under the `pages/Task` folder with the other tables!